vizro-ai/hatch.toml

```
@@ -0,0 +1,67 @@
+[envs.all]
+
+[[envs.all.matrix]]
+python = ["3.9", "3.10", "3.11"]
+
+[envs.changelog]
+dependencies = ["scriv"]
+detached = true
+scripts = {add = "scriv create --add"}
+
+[envs.default]
+dependencies = [
+  "devtools[pygments]",
+  "pre-commit",
+  "coverage[toml]>=6.5",
+  "pytest",
+  "toml",
+  "nbformat>=4.2.0"
+]
+
+[envs.default.env-vars]
+VIZRO_AI_LOG_LEVEL = "DEBUG"
+
+[envs.default.scripts]
+cov = [
+  "test-cov",
+  "cov-report"
+]
+cov-report = [
+  "- coverage combine",
+  "coverage report"
+]
+lint = "SKIP=gitleaks pre-commit run {args} --all-files"
```

```suggestion
lint = "SKIP=gitleaks pre-commit run {args:--all-files}"
```
as suggested here: https://github.com/mckinsey/vizro/pull/136/files#diff-2c756b1ed57164797d707fa9b6ee28c08870236a60733efa77525584b1afec8dR44